# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1506059514 25200
#      Thu Sep 21 22:51:54 2017 -0700
# Node ID 1457c018b6836985225406b4aeea72efa7292675
# Parent  0052d47ddc3e082ce7a952754eb44bcc93db9e95
maint: declare gzip compression options to stay compatible with Automake 1.14

* Makefile.am (GZIP_ENV): Quote the list of options for compatibility with
Automake 1.14.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -78,17 +78,17 @@ OCT_LINK_OPTS = @OCT_LINK_OPTS@
 LIBOCTGUI_LINK_DEPS = @LIBOCTGUI_LINK_DEPS@
 LIBOCTGUI_LINK_OPTS = @LIBOCTGUI_LINK_OPTS@
 
 OCTAVE_GUI_LINK_DEPS = @OCTAVE_GUI_LINK_DEPS@
 OCTAVE_GUI_LINK_OPTS = @OCTAVE_GUI_LINK_OPTS@
 
 # Options used for creating the source distribution.
 
-GZIP_ENV = --best --no-name
+GZIP_ENV = '--best --no-name'
 SOURCE_MTIME := \
   $(shell $(SHELL) $(top_srcdir)/build-aux/get-source-mtime.sh "$(srcdir)")
 TAR_OPTIONS = $(REPRODUCIBLE_TAR_FLAGS) --mtime=@$(SOURCE_MTIME)
 
 # The arguments passed to configure.
 
 CONFIG_SUBDIRS = @subdirs@
 
